src/components/UserCard/PartnerCard.tsx

```diff
@@ -0,0 +1,64 @@
+import IconLink from '@material-ui/icons/Link'
+import IconMail from '@material-ui/icons/Mail'
+import IconPhone from '@material-ui/icons/Phone'
+import React from 'react'
+import { Avatar } from 'src/components/Avatar'
+import { ExtendedUserPartner } from 'src/core/api'
+
+import * as S from './UserCard.styles'
+
+interface PartnerCard {
+  onClickBack: () => void;
+  partner?: ExtendedUserPartner;
+}
+
+export function PartnerCard(props: PartnerCard) {
+  const name = props?.partner?.name
+  const smallLogoUrl = props?.partner?.smallLogoUrl
+  const description = props?.partner?.description
+  const email = props?.partner?.email
+  const phone = props?.partner?.phone
+  const websiteUrl = props?.partner?.websiteUrl
+  const donationsNeeds = props?.partner?.donationsNeeds
+  const volunteersNeeds = props?.partner?.volunteersNeeds
```

Comment:
l'objet props ne sera jamais `undefined`, du coup pas besoin de `?` juste derrière.
D'ailleurs c'est un peu redondant toutes ces lignes, on devrait pouvoir destructurer comme ça:

```suggestion
  const {
    name,
    smallLogoUrl,
    description,
    email,
    phone,
    websiteUrl,
    donationsNeeds,
    volunteersNeeds
  } = props.partner || {}
```